tensorflow/contrib/tensor_forest/client/random_forest_test.py

```diff
@@ -82,6 +99,27 @@ def testClassification(self):
     self.assertAllClose([[0.576117, 0.211942, 0.211942]],
                         [pred['probabilities'] for pred in predictions])
 
+  def testClassificationDeterministic(self):
+    """Test multi-class classification is deterministic"""
+    hparams = tensor_forest.ForestHParams(
+        num_trees=3,
+        max_nodes=1000,
+        num_classes=3,
+        num_features=4,
+        split_after_samples=20,
+        inference_tree_paths=True)
```

Comment:
should not it have a seed? i thought determinism will be on only when the seed is provided